### ISSUE TITLE
[BUG] In Chromium using fill on textarea with long text, takes more than 5 seconds (and destroys something in background)

### ISSUE THREAD

**Issue by gladykov** [microsoft/playwright#23077]
### System info
- Playwright Version: v1.33
- Operating System: Endevour OS (Kernel 6.3) + Ubuntu 22.04.2 LTS 
- Browser: Chromium

### Source code
[fill.zip](https://github.com/microsoft/playwright/files/11493217/fill.zip)

**Steps**
- In runMe.ts setup `root` variable with  path, where you extracted files
- Run

**Expected**

- Textarea on `index.html` is filled immediately after file contents are read into memory

**Actual**

- It takes 7 seconds on my laptop to fill it with 28KB of text

And now the funny part, for which I do not have test case: Above creates some unexpected state, which causes failure of web page elements (in publishing Confluence page flow):
- clicking Publish will cause web page error "Unable to publish"
- there is no clear indication of failure in console
- network requests are firing
- looks like regression in Confluence software, as it worked properly for one month
- Still bug exists ONLY when using playright fill method + big text  

**Notes**

- Both above bugs will not occur when using shorter text with fill method
- When same text will be pasted (Ctrl+V) manually, during open playwright browser session, publish action will work properly (so it is not caused by combination of browser flags and web page)
- Our real world scenario use case uses iframes
- Using `type` method to type 28KB in textarea had even more destructive effect, as other buttons of UI stopped working (but not all of them, which is again something very peculiar)

TL:DR; Long text takes too long to fill textarea when using fill method and I suspect it creates some overflow state, which brakes some UI elements bindings; Using type method broke even more.

**Comment by dgozman** [comment]
I can reproduce. `Input.insertText` CDP command takes very long time. Needs debugging upstream.

**Comment by dgozman** [comment]
Upstream issue: https://bugs.chromium.org/p/chromium/issues/detail?id=1451039.

**Comment by pavelfeldman** [comment]
#### Why was this issue closed?

Thank you for your contribution to our project. This issue has been closed due to its limited upvotes and recent activity, and insufficient feedback for us to effectively act upon. Our priority is to focus on bugs that reflect higher user engagement and have actionable feedback, to ensure our bug database stays manageable.

Should you feel this closure was in error, please create a new issue and reference this one. We're open to revisiting it given increased support or additional clarity. Your understanding and cooperation are greatly appreciated.

**Comment by gladykov** [comment]
Workaround for headless mode on Linux, by using Ctrl + V : https://github.com/microsoft/playwright/issues/8114#issuecomment-2174229844

**Comment by pavelfeldman** [comment]
#### Why was this issue closed?

Thank you for your contribution to our project. This issue has been closed due to its limited upvotes and recent activity, and insufficient feedback for us to effectively act upon. Our priority is to focus on bugs that reflect higher user engagement and have actionable feedback, to ensure our bug database stays manageable.

Should you feel this closure was in error, please create a new issue and reference this one. We're open to revisiting it given increased support or additional clarity. Your understanding and cooperation are greatly appreciated.
